src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -1198,8 +1199,11 @@ public function toggleStatusAction($productId)
 
         try {
             $isEnabled = $this->getQueryBus()->handle(new GetProductIsEnabled($productId));
-
-            $this->getCommandBus()->handle(new UpdateProductStatusCommand($productId, !$isEnabled));
+            $this->getCommandBus()->handle(new UpdateProductStatusCommand(
```

Comment:
Ah I forgot that this API as on the legacy controller We need to move/copy this in the V2 controller I know it's not the initial scope of this PR but it's a good occasion to do it here, what do you think?

Reply:
Nah, its not in scope. I have one little well scoped task and im ok with it.
Those "innocent little changes" accumulates and I end up having 3k lines PR which needs rebases 2times a week. No thanks, I'll pass :sweat_smile: